app/src/main/res/layout/fragment_movie_list.xml

```diff
@@ -1,12 +1,192 @@
 <?xml version="1.0" encoding="utf-8"?>
 
+<androidx.constraintlayout.widget.ConstraintLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@color/background"
+    tools:context=".UI.MovieListFragment">
+
+    <Button
+        android:id="@+id/buttonViewMenu"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="@dimen/margin_star_of_parent"
+        android:layout_marginTop="44dp"
+        android:background="@android:color/transparent"
+        android:drawableLeft="@drawable/ic_combined_shape"
+        android:gravity="center"
+        android:text="@string/movies_list"
+        android:textAppearance="@style/button_back"
+        android:textSize="@dimen/body_text_size"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
     <androidx.constraintlayout.widget.ConstraintLayout
```

Comment:
Can we do it with one constraint layout only?

Reply:
We can use cardView instead of a constrainLayout because in any case we need to circle the boundary of the elements.